browser/profiles/brave_profile_impl.cc

```diff
@@ -56,7 +56,7 @@ BraveProfileImpl::BraveProfileImpl(
     notification_registrar_.Add(this, chrome::NOTIFICATION_PROFILE_DESTROYED,
                                 content::Source<Profile>(parent_profile));
     base::PostTaskAndReply(
-        FROM_HERE, base::DoNothing(),
+        FROM_HERE, {base::ThreadPool()}, base::DoNothing(),
```

Comment:
this is not correct (and was also not correct before). This should be posting to the UI thread

Reply:
Addressed by 3e9dede3537553d72fa9d29cc9c02b8c11a8757f